README.md

```diff
@@ -186,9 +186,6 @@ There are three artifacts with extra support for Android:
 * `autodispose-android-archcomponents` has a `AndroidLifecycleScopeProvider` for use with `LifecycleOwner` and `Lifecycle` implementations.
 * `autodispose-android-archcomponents-test` has a `TestLifecycleOwner` for use in testing.
 
-For each artifact, there is a corresponding kotlin extensions artifact with it. Example:
```

Comment:
Let's tweak this to just say it's WIP for next release and link the ticket

Reply:
6e96d95